charts/irs-helm/README.md

```diff
@@ -1,16 +1,18 @@
+[[Back to main README](../../README.md)]
+
 # Item Relationship Service Helm Chart
 
 This chart installs the Item Relationship Service and its direct dependencies.
 This includes:
 
- - Minio Object Storage (enabled by default)
- - Prometheus (disabled by default) 
- - Grafana (disabled by default) 
+ - [Minio Object Storage](https://min.io) (enabled by default)
+ - [Prometheus](https://prometheus.io) (disabled by default) 
+ - [Grafana](https://grafana.com/grafana) (disabled by default) 
 
 ## Prerequisites
 
-- Kubernetes 1.19+
-- Helm 3.2.0+
+- [Kubernetes](https://kubernetes.io) 1.19+
+- [Helm](https://helm.sh) 3.2.0+
```

Comment:
Please remove version information here  - therefor the compatibility matrix and the project is used. This might be outdated soon.

Reply:
see answer above

Reply:
see answer  above

Reply:
Could you point me to the answer, please.

Reply:
https://github.com/eclipse-tractusx/item-relationship-service/pull/456#discussion_r1511493578